packages/tabs/src/elements/Tabs.tsx

```diff
@@ -17,16 +17,16 @@ import { StyledTabs } from '../styled/StyledTabs';
 
 export interface ITabsProps extends Partial<ThemeProps<DefaultTheme>> {
   /**
-   * Toggles vertical selection mode of Tabs
+   * Displays the child `Tab` components in a vertical orientation
    */
   isVertical?: boolean;
   /**
-   * Currently selected tab to display
+   * Sets the currently selected `Tab` to display
    */
   selectedItem?: any;
   /**
-   * Callback for when a tab has been selected by keyboard or mouse
-   * @param {String} selectedItem - The item of the selected tab
+   * Provides a callback function that returns the `selectedItem` when a `Tab` has been selected by keyboard or mouse
+   * @param {String} selectedItem: The `item` value of the selected `Tab`
```

Comment:
What happens with this syntax? Usually, proper jsdoc syntax would be `@param {String} name Description...` with no intervening colon or dash – just space.

Reply:
The thinking here was that the sentence following the parameter seemed like an explanation/definition of the word, so a colon seemed appropriate. I suppose since I changed it my preference is colon but dash works too. Was thinking having nothing to separate the word from the explanation might look like one complete sentence.

Reply:
I pulled your branch. We should drop the invalid `:` and worry about structural output of the data when it renders on `website`.

Reply:
Valid jsdoc here is 

```js
@param {String} selectedItem The `item` value of the selected `Tab`
```

Reply:
Perfect, ty